src/main/java/net/sf/jabref/logic/util/io/FileHistory.java

```diff
@@ -49,9 +49,9 @@ public int size() {
 
     public void newFile(String filename) {
         history.remove(filename);
-        history.addFirst(filename);
+        ((LinkedList<String>) history).addFirst(filename);
         while (history.size() > HISTORY_SIZE) {
-            history.removeLast();
+            ((LinkedList<String>) history).removeLast();
```

Comment:
Indeed. Should one keep the definition as a LinkedList then? (I have to cast it to get access to removeLast().)